ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -27,7 +27,14 @@ import SwiftUI
 ///
 @objc public class MSFListCellState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
-    @objc @Published public var leadingView: UIView?
+    @Published public var leadingUIView: AnyView?
```

Comment:
Can you also update the documentation comments of the state class? (please make sure to emphasize that setting the UIView property overwrites the AnyView property)